backend/src/sms/services/sms-transactional.service.ts

```diff
@@ -31,9 +41,70 @@ async function sendMessage({
     message: 'Sending transactional SMS (service)',
     action: 'sendMessage',
   })
+
+  // append the config callback info into credentials here
+  credentials.callbackSecret = config.get('smsCallback.callbackSecret')
+  credentials.callbackBaseUrl = config.get('smsCallback.callbackUrl')
   return await SmsService.sendMessage(credentials, recipient, sanitizedBody)
 }
 
+async function listMessages({
+  userId,
+  limit,
+  offset,
+  sortBy,
+  orderBy,
+  status,
+  filterByTimestamp,
+}: {
+  userId: string
+  limit?: number
+  offset?: number
+  sortBy?: TransactionalEmailSortField
```

Comment:
Should create a separate `TransactionalSmsSortField`

Reply:
done